### PR TITLE
Configuration simplifications

### PR DESCRIPTION
**Pull request by jbygdell**
**This needs to go in on top of #104**

## Changes made
* Secrets for the certificates are changed from `Opaque` to `tls`, this makes it possible for things like `Cert-manager` to create the certificates.
* Ca certificate can be manually created as a secret or added to the values file
* Init containers are used for the java components to generate the required java specific certificates
* Certificates are now created with proper `/CN` tag that enables full peer verification by PostgreSQL and RabbitMQ
* Added a script to cleanup all deployed parts
* Fixes to the helm test script

### CODE REVIEW

File: charts/sda-svc/test/trigger-ingest.py [hunk 1]
```diff
@@ -47,13 +47,18 @@
 
 if mq_connection.startswith('amqps'):
```
Comment (blankdots) [inline charts/sda-svc/test/trigger-ingest.py:48]: i think we are doing a double check here `if tls == 'true'` and lower we check if mq.connection starts with `amqps` which is basically the same thing, maybe we can simplify that (not sure we will ever have `tls = true` and `amqp` protocol.

File: charts/sda-svc/test/trigger-ingest.py [hunk 2]
```diff
@@ -63,11 +68,12 @@
 
     # If client verification is required
     if certfile.exists():
-        assert( keyfile.exists() )
+        assert(keyfile.exists())
```
Comment (blankdots) [inline charts/sda-svc/test/trigger-ingest.py:71]: not ideal maybe we can check `if certfile.exists() and keyfile.exists():` if not we raise execption or do an sys.exit() and an error code from: https://docs.python.org/3.8/library/os.html#os._exit

**Review by blankdots — COMMENTED** [review]
if `{{ template "tlsPath" . }}/ca.crt` and `{{ template "tlsPath" . }}/tls.crt` + `{{ template "tlsPath" . }}/tls.key` are the same for most services. we can use variables set from `_helper`.
The only downside to that would be that they will be somewhat hidden if one would try to find them.

File: charts/sda-svc/test/trigger-ingest.py (hunk 1 above)
Comment (jbygdell) [reply charts/sda-svc/test/trigger-ingest.py:48]: I touched as little as possible in the test suite.
As for the python stuff, it needs to be rewritten so we can test everything properly, even in a complete standalone installation with the orchestrator.

Comment (blankdots) [reply charts/sda-svc/test/trigger-ingest.py:48]: fair enough, then we can have that as part of another PR

File: charts/sda-svc/test/trigger-ingest.py (hunk 2 above)
Comment (blankdots) [reply charts/sda-svc/test/trigger-ingest.py:71]: fair enough, then we can have that as part of another PR

**Review by blankdots — APPROVED** [review]
`tls.{crt|key}` could be added as part of `_helpers`, and python scripts can be done in different PRs
approved as is

**Review by dbampalikis — APPROVED** [review]

**Review by norling — APPROVED** [review]

**Review by blankdots — APPROVED** [review]

**Pull request merged**